core/src/main/scala/com/raphtory/api/analysis/graphview/DeployedTemporalGraph.scala

```diff
@@ -30,14 +30,12 @@ class DeployedTemporalGraph private[raphtory] (
   def config: Config = conf
 
   def destroy(force: Boolean = false): Unit = {
-    if (!local)
-      querySender.destroyGraph(force)
+    querySender.destroyGraph(force)
     shutdown.unsafeRunSync()
   }
 
   override def close(): Unit = {
-    if (!local)
-      querySender.disconnect()
+    querySender.disconnect()
```

Comment:
We need to remember to make this `close` idempotent

Reply:
Probably will add a comment for this PR. Will certainly consider this as part of next PR. Adding this to: **Are there any further changes required?**

Reply:
I just checked the `OrchestratorComponent`, it logs **WARN** if graph is not found when destroying and disconnecting graphs.

```
logger.warn(s"Graph '$graphID' requested for destruction by '$clientID', but did not exist")
logger.warn(s"'$clientID' disconnected from Graph '$graphID', but did not exist")
```

Which makes these APIs idempotent!